src/server.js

```diff
@@ -40,11 +43,24 @@ function calculateReward(value, volume) {
     return "Doesn't even fit in an Orca!";
   };
 
+  // If someone pastes in units of millions of ISK, convert to units of ones of ISK.
+  // 20.25 -> 20,250,000
+  if (value < 100000) {
```

Comment:
I'm used to typing in "120" (million) or w/e instead of copy pasting some really long string from our old calculator and it's very convenient.